build.gradle

```diff
@@ -54,39 +54,12 @@ uploadArchives {
         mavenDeployer {
             beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-                authentication(userName: ossrhUsername, password: ossrhPassword)
+            repository(url: "https://services.api.bskyb.com/nexus/content/repositories/sky-releases/") {
+                authentication(userName: archivaUser, password: archivaPass)
```

Comment:
You can't put your internal repo on a public project. 

Make them properties and default them at the top like ossrhUsername to maven central